style.css

```diff
@@ -429,6 +432,108 @@ fieldset textarea {
   }
 }
 
+/* Popup window -----------------------------------> */
+.modal {
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  background-color: rgba(255, 255, 255, 0.16);
+  transform: translate(-50%, -50%) scale(0);
+  transition: 200ms ease-in-out;
+  border: 1px solid white;
+  z-index: 10;
+  width: 327px;
+  height: 637px;
+}
```

Comment:
- [ ] Kindly double check on the styling of the modal on the mobile version and fix the overflowed space on some widths and heights.

- [ ] Also, please keep in mind to follow the given design in (typography, font size, weight, colors, images, and spacings)

_Please take a look at the two images down below, then apply the corresponding changes to match the template._

### **Your design**
![Screenshot](https://user-images.githubusercontent.com/47625407/123021244-04ae6e80-d3d4-11eb-8821-3f5122986c4f.png)

### **[Original design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=37%3A1288)**
![Screenshot1](https://user-images.githubusercontent.com/47625407/123021279-155ee480-d3d4-11eb-9258-aefcd6c22322.png)

Reply:
I've tried to put like the specifications on Figma but it is getting overflow this `  width: 327px;
  height: 847px;`  
![Screen Shot 2021-06-23 at 15 20 32](https://user-images.githubusercontent.com/67211919/123163236-1bd27800-d437-11eb-9a12-9319ad45f60f.png)